resources/user-agents/browsers/mercury/mercury-generic.json

```diff
@@ -1,17 +1,17 @@
 {
-  "division": "Mercury",
+  "division": "Mercury Generic",
```

Comment:
I added "Generic" here to be more consistent with other files, this did require changing some tests.